Code/Diversity_Histogram.py

```diff
@@ -5,25 +5,30 @@
 This calculation requires the following data files: 
     Overlap_First_Interactors_NoDuplicates.xlsx
 
+
 """
 
 
 import pandas as pd
+import numpy as np
 import seaborn as sns
 import scipy
 from scipy.stats import norm
+#%% Import interaction matrix (in which each interactor occurs only once)
+Interaction_Matrix = pd.read_excel (r'Code/Data_Files/Overlap_First_Interactors_NoDuplicates.xlsx')
```

Comment:
I believe the addition of 'Code/' should be removed since that would not be the correct path